cratedb_toolkit/admin/xmover/cli.py

```diff
@@ -62,6 +62,16 @@ def analyze(ctx, table: Optional[str]):
     reporter.distribution(table=table)
 
 
+@main.command()
+@click.pass_context
+def monitor_shards(ctx):
+    """Monitor shards, pointing out hot ones"""
+    client = ctx.obj["client"]
+    analyzer = ShardAnalyzer(client)
+    monitor = ShardMonitor(analyzer)
+    monitor.monitor_shards()
```

Comment:
Just an idea about »naming things«: `xmover monitor-shards` certainly fits the bill, but what do you think about `xmover shard-heat` [^1] to better reflect its intentions?

[^1]: ... with an optional `--watch` option in the long run?